Fabric.Terminology.API/Fabric.Terminology.API.csproj

```diff
@@ -43,9 +43,10 @@
   <ItemGroup>
     <None Remove="Fabric.Terminology.API.ruleset" />
     <None Remove="FodyWeavers.xml" />
-    <None Remove="appsettings.dev.json" />
+    <Content Remove="appsettings.dev.json" />
```

Comment:
these changes were needed to exclude the appsettings.dev.json file to be excluded from the publish